app.js

```diff
@@ -1,474 +1,570 @@
-var token = ''
+var token = "";
 
 // the backdrop
 var backdrop = $('<div class="kamino-backdrop fade in"></div>');
 
 // repo list
-var repoList = []
+var repoList = [];
```

Comment:
`;` are not something I want in the code files for this project. Please remove all instances of `;`